Core/ComponentBaseClasses/elxTransformBase.h

```diff
@@ -374,7 +370,7 @@ class TransformBase :
   }
 
   /** Boolean to decide whether or not the transform parameters are written in binary format. */
-  bool m_UseBinaryFormatForTransformationParameters;
+  bool m_UseBinaryFormatForTransformationParameters{};
```

Comment:
should this not default to false? like in the current constructor?

Reply:
Thanks for asking, Marius.
> should this not default to false?
 Indeed, and it does! `{}` is the default way to initialize a variable in C++11.  For an integer type, that is zero, for a `bool`, it is `false`. OK?

Reply:
clear!